hack/bumper.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Package bump helper
+# Make sure you already have a repository (i.e CNAO) with remote upstream, and origin branches
+# No untracked files are allowed on folder (they will be added and pushed, so make sure folder is clean)
+
+# ./hack/bumper.sh CVE-2021-38561 golang.org/x/text@v0.3.7 release-0.89
+
+if [ $# -ne 3 ]; then
+    echo "Syntax: $0 <CVE> <TARGET_PACK> <BR>"
+    exit 1
+fi
+
+CVE=$1
+TARGET_PACK=$2
+BR=$3
+
+PACK="${TARGET_PACK/@*}"
+git checkout "${BR}"
+git pull upstream "$(git symbolic-ref --short HEAD)"
+go mod edit -dropreplace="${PACK}"
+go mod edit -require="${TARGET_PACK}"
+make vendor
```

Comment:
```suggestion
make fmt vet
```
make fmt already does vendor
make vet checks bump didn't brake anything

Reply:
this script is meant to be generic, not just for CNAO, 
i am not sure we have on all our repos fmt / vet
(we dont even have vendor on all repos, i had a better version that handled this case)

so i prefer to not add it now, before we make sure it works all over
whoever needs can just run the additonal commands and amend
more info / options here https://github.com/kubevirt/cluster-network-addons-operator/pull/1968#issuecomment-2574607216

atm i would prefer to not block due to that

Reply:
But we should fail before the PR is created, 
how about adding `go vet $WHAT` where WHAT is defaulted to `./pkg/...`?

Reply:
Even if we don't fail and just run later commit amend it is fine basically (of course we can optimize)
i am against adding something like `go vet $WHAT`, because on some repos we don't run go vet.
What i can do is add make vet if make vet target exists, this way it is generic and robust, as this script is intended